src/lib/ecr_interface.py

```diff
@@ -0,0 +1,23 @@
+from boto3 import client as boto3client
+
+class ECRInterface(object):
+  ''' interface with AWS ECR '''
+
+  def __init__(self):
+    self._ecr = boto3client('ecr')
```

Comment:
Unconventional var name here... use either snake case or for a class the convention is cap-case `Boto3Client`.

Reply:
We usually just use `boto3.client()` though and access it like a packaged function.

Reply:
[snake case'd](https://github.com/artsy/opstools/commit/89091a30ae0720600720b9223af34f23e3e87efa)

Reply:
Ah I see